radixdlt/src/main/java/com/radixdlt/network/messaging/MessageDispatcher.java

```diff
@@ -161,6 +138,54 @@ void receive(MessageListenerList listeners, final MessageEvent inboundMessage) {
 		this.counters.increment(CounterType.MESSAGES_INBOUND_PROCESSED);
 	}
 
+	private Peer handleSystemMessage(Peer oldPeer, SystemMessage systemMessage) {
```

Comment:
Restructured to separate out concerns.  Now checks signatures on signed messages.